docs/source/cpp/compute.rst

```diff
@@ -280,6 +284,10 @@ an ``Invalid`` :class:`Status` when overflow is detected.
 +--------------------------+------------+--------------------+---------------------+
 | subtract_checked         | Binary     | Numeric            | Numeric             |
 +--------------------------+------------+--------------------+---------------------+
+| negate                   | Unary      | Numeric            | Numeric             |
++--------------------------+------------+--------------------+---------------------+
+| negate_checked           | Unary      | Numeric            | Numeric             |
++--------------------------+------------+--------------------+---------------------+
```

Comment:
These tables are alphabetically-ordered, it would be nice to keep them like that.